src/sentry/static/sentry/app/components/events/interfaces/spans/spanDetail.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import styled from 'react-emotion';
+import {get, map} from 'lodash';
+
+import DateTime from 'app/components/dateTime';
+import Pills from 'app/components/pills';
+import Pill from 'app/components/pill';
+import space from 'app/styles/space';
+
+import {SpanType} from './types';
+
+type PropTypes = {
+  span: Readonly<SpanType>;
+};
+
+const SpanDetail = (props: PropTypes) => {
+  const {span} = props;
+
+  const startTimestamp: number = span.start_timestamp;
+  const endTimestamp: number = span.timestamp;
+
+  const duration = (endTimestamp - startTimestamp) * 1000;
+  const durationString = `${duration.toFixed(3)} ms`;
+
+  return (
+    <SpanDetailContainer
+      data-component="span-detail"
+      onClick={event => {
+        // prevent toggling the span detail
+        event.stopPropagation();
+      }}
+    >
+      <table className="table key-value">
```

Comment:
Are these bootstrap classNames? It would be nice to have them as styled components

Reply:
I just copied this table format from the issues page.

There was no authoritative design on the span details; so I recycled and cobbled something together to present the span data.